app/controllers/export_controller.rb

```diff
@@ -33,6 +33,8 @@ def tei
     @user_contributions =
       User.find_by_sql("SELECT  user_id user_id,
                                 users.real_name real_name,
+                                users.display_name display_name,
+                                users.guest guest,
```

Comment:
Why do we have guests here?  I suspect that would leave us with some "guest" IDs in the attributions, which I'm not sure we want.